service/portability-service-lambda/portability_service.py

```diff
@@ -0,0 +1,330 @@
+"""Service to run workflows in different environments, demonstrating their
+portability.
+"""
+import base64
+import datetime
+import io
+import json
+import os
+import uuid
+import zipfile
+
+import boto3
+from boto3.dynamodb.conditions import Key
+import requests
+
+#
+# NB: Mentions of "WES" in this file are aspirational. Nothing here quite
+# matches the currently published GA4GH WES spec, but that's a goal.
+#
+
+# Set of environment variables passed here from the Cloudformation template.
```

Comment:
Where does this Cloudformation template live? Is it possible to link to it here?

Reply:
This is `portability_service_template.json`, adjacent to this script. I can clarify that.

Reply:
Oh, I see! Yes that would be clearer!